docs/SPEC.md

```diff
@@ -2172,13 +2173,15 @@ Sort has the following properties:
 | desc    | bool     | Desc indicates results should be sorted in descending order.                              |
```

Comment:
Can you put the default value for `desc`?

Reply:
Done.